assets/src/edit-story/components/inspector/prepublish/test/prepublishChecklistProvider.js

```diff
@@ -62,29 +63,32 @@ const generateStoryPages = (pageCount) => {
 };
 
 function setup({ pageCount = 1 }) {
-  const storyContext = {
-    actions: {},
-    state: {
-      currentPage: {},
-      pages: [],
-      story: {
-        featuredMedia: {
-          url: 'https://greatimageaggregate.com/1234',
-        },
-        title: 'How to get rich',
-        excerpt: "There's a secret no one wants you to know about",
-        author: { id: 1, name: 'admin' },
-        status: 'draft',
-        pages: generateStoryPages(pageCount),
+  const fullStory = {
+    currentPage: {},
+    pages: [],
+    story: {
+      featuredMedia: {
+        url: 'https://greatimageaggregate.com/1234',
       },
+      title: 'How to get rich',
+      excerpt: "There's a secret no one wants you to know about",
+      author: { id: 1, name: 'admin' },
+      status: 'draft',
+      pages: generateStoryPages(pageCount),
     },
   };
+  const storyContext = {
+    actions: {},
+    state: fullStory,
+  };
 
   const wrapper = ({ children }) => (
     <StoryContext.Provider value={storyContext}>
-      <LayoutProvider value={layoutContext}>
-        <PrepublishChecklistProvider>{children}</PrepublishChecklistProvider>
-      </LayoutProvider>
+      <StoryTriggersProvider story={fullStory}>
+        <LayoutProvider value={layoutContext}>
+          <PrepublishChecklistProvider>{children}</PrepublishChecklistProvider>
+        </LayoutProvider>
+      </StoryTriggersProvider>
```

Comment:
the below test is now actually working so it should actually expect `UNAVAILABLE` not `ALL`